client/my-sites/hosting/main.js

```diff
@@ -177,6 +180,7 @@ class Hosting extends Component {
 			const isGithubIntegrationEnabled = isAutomatticTeamMember( teams );
 			const WrapperComponent = isDisabled || isTransferring ? FeatureExample : Fragment;
 			const isStagingSiteEnabled = isEnabled( 'yolo/staging-sites-i1' );
+			const { hasStagingSites } = this.props;
```

Comment:
Would it be better to destruct it upper in the stack, as a part of `render()` where we destruct other props?